extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -81,6 +81,8 @@ function render_player( $track_list, $attributes ) {
 	if ( empty( $track_list ) ) {
 		return '<p>' . esc_html__( 'No tracks available to play.', 'jetpack' ) . '</p>';
 	}
+	static $counter = 0;
+	$instance_id    = FEATURE_NAME . '-' . ( $counter++ );
```

Comment:
Instead of a counter, we've opted to rely on `wp_unique_id` for the Calendly block:
https://github.com/Automattic/jetpack/blob/0365248305076dfa0f6f8bceb89e4ec9fed18d1f/extensions/blocks/calendly/calendly.php#L115
Maybe we could do the same here?

Reply:
Good idea, will do